config/routes.rb

```diff
@@ -8,11 +8,13 @@
   # route /apps/:app_id/engagements/:engagement_id
   resources :apps do
     resources :engagements, :except => :index
+    resources :engagements, :only => :destroy, :as => 'engagement_destroy'
```

Comment:
Was there any reason why you changed the path names? Seems like we already have :app_engagement_path and :engagement_iteration_path

Reply:
There are indeed existing paths but they did not have a 'prefix' like edit, create, index, etc. when I ran "rake routes". I just created a prefix with the same path to be more readable in the link_to call.

Reply:
oh, they all share `app_engagement`. Just use it with different HTTP method.
### Example ###
* app_comment + get => comments#show
* app_comment + delete => comments#destroy

Reply:
Ohh yes I see. There was no need to create a prefix. I just changed the method and it worked. Thanks Sean! Pushing the changes now